Week-1/assignment_2/index.css

```diff
@@ -0,0 +1,121 @@
+* {
+  padding: 0;
+  margin: 0;
+  font-family: "Lato";
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  width: 100vw;
+  height: 90vh;
+}
+
+.sidebar {
+  background-color: #f0f0f0;
+  width: 3vw;
+  height: 90vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.body {
```

Comment:
we can wrap in a div and add styles to it.. if there is any other requirement it might break the ui